### PR TITLE
Iterate files in predictable order

### PR DESCRIPTION
**Pull request by thockin**
This helped me debug a different issue.  Diffing logs is unfriendly without it.

### CODE REVIEW

File: parser/parse.go [hunk 1]
```diff
@@ -425,13 +426,20 @@ func (b *Builder) typeCheckPackage(pkgPath importPathString) (*tc.Package, error
 // FindPackages fetches a list of the user-imported packages.
 // Note that you need to call b.FindTypes() first.
 func (b *Builder) FindPackages() []string {
+	// Iterate packages in a predictable order.
+	keys := []string{}
```
Comment (sttts) [inline parser/parse.go]: nit: s/keys/pkgPaths/

Comment (sttts) [reply parser/parse.go]: also below

**Comment by sttts** [comment]
Some naming nit. Otherwise lgtm.

**Comment by thockin** [comment]
Updated

**Comment by thockin** [comment]
Is this repo bot-merged now?  It used to just be manual.

**Comment by sttts** [comment]
Looks like. Feel free to click, I can't.

**Comment by thockin** [comment]
we shoudl fix that

**Pull request merged**